sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -196,6 +197,17 @@ object DateTimeUtils extends SparkDateTimeUtils {
     localDateToDays(daysToLocalDate(days).plusMonths(months))
   }
 
+  /**
+   * Returns the three-letter abbreviated month name for the given number of days 1970-01-01.
+   */
+  def getMonthName(days: Int): UTF8String = {
+    val monthName = Month
+      .of(getMonth(days))
+      .getDisplayName(TextStyle.SHORT, Locale.US)
```

Comment:
Could you use `DateFormatter.defaultLocale`, please. (at least we don't spread the default local settings across source code).

Reply:
makes sense!